src/Tezos/Forging.cpp

```diff
@@ -42,11 +76,31 @@ Data forgePublicKeyHash(const std::string& publicKeyHash) {
         throw std::invalid_argument("Invalid Prefix");
     }
     const auto decoded = Base58::bitcoin.decodeCheck(publicKeyHash);
-    const auto prefixSize = 3;
+    constexpr auto prefixSize{3};
     forged.insert(forged.end(), decoded.begin() + prefixSize, decoded.end());
     return forged;
 }
 
+Data forgeAddress(const std::string& address) {
+    if (address.size() < 3) {
+        throw std::invalid_argument("Invalid address size");
+    }
+    auto prefix = address.substr(0, 3);
+    if (prefix == "tz1" || prefix == "tz2" || prefix == "tz3") {
+        Data forged{0x00};
+        append(forged, forgePublicKeyHash(address));
+        return forged;
+    } else if (prefix == "KT1") {
+        Data forged{0x01};
+        const auto decoded = Base58::bitcoin.decodeCheck(address);
```

Comment:
Error check is needed here?  It may return {} in case of error

Reply:
code was duplicated, and not checked before, cc @hewigovens - I've made a common function used in both function now, but still without any check.

Reply:
lgtm